go.mod

```diff
@@ -1,36 +1,31 @@
 module github.com/spacelift-io/vcs-agent
 
-go 1.23
+go 1.24
 
 require (
 	github.com/bugsnag/bugsnag-go/v2 v2.5.1
-	github.com/franela/goblin v0.0.0-20211003143422-0a4f594942bf
```

Comment:
When I was reading the PR at first glance, I didn't notice that it was for the vcs agent and I was super surprised that we got rid of those for the backend repo already 😄 
Anyway, thanks for that ❤️

Reply:
one day :)